pkg/lv_drivers/include/display/monitor_disp_dev.h

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (C) 2019 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_monitor
```

Comment:
I find "monitor" a somewhat unlucky name. "screen", but then where's the difference to an embedded display. "sdl"?

Reply:
It mimics the name use in lv-drivers

Reply:
well, this got renamed to "sdl" in lv_drivers >8.0. Lets keep it here.